Build.md

```diff
@@ -2,100 +2,194 @@
-        - **Windows XP support for C++**
-  - For Visual Studio 2019:
-    - In the Visual Studio Installer, you'll need to install:
-        - Workloads
-          - **Desktop development with C++**
-  - Download and install [Qt](https://www.qt.io/download) for MSVC 2017 32-bit
-  - Download and install [CMake](http://www.cmake.org) for Windows
-  - Download and install [pandoc](http://www.pandoc.org)
-  - Create a subdirectory of the TrenchBroom directory, e.g. "build", and open a command prompt there
-  - For Visual Studio 2017, run:
-
-    ```
-    cmake .. -T v141 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=C:\Qt\5.13.0\msvc2017
-    ```
-
-  - For Visual Studio 2019, run:
-
-    ```
-    cmake .. -G "Visual Studio 16 2019" -A Win32 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH=C:\Qt\5.13.0\msvc2017
-    ```
-
-    Finally, open `build\TrenchBroom.sln`.
-
-  - **Note:** specifying CMAKE_BUILD_TYPE is required when running cmake; you can't change between Release and Debug from Visual Studio
+Visual Studio 2017 or 2019 can be used for development. TrenchBroom needs to compile with VS 2017 (that's what our releases are built with).
+
+### Prerequisites
+
+In order to develop, debug and compile TrenchBroom, you need to install tools listed below.
+
+#### Visual Studio 2017
+
+Download [VS 2017](https://visualstudio.microsoft.com/vs/older-downloads/#visual-studio-2017-family) and install it.
+
+You'll also need to install these dependencies using Visual Studio Installer (included with main installer):
+
+- Workloads 
+  - **Desktop development with C++**
+- Individual components
+  - **C++ Windows XP Support for VS 2017 (v141) tools**
+  - **MSVC v141 - VS 2017 C++ x64/x86 build tools (v14.16)**
+  - **Windows Universal CRT SDK**
+  
+#### Visual Studio 2019
+
+Download [VS 2019](https://visualstudio.microsoft.com/vs/) and install it.
+
+You'll also need to install these dependencies using Visual Studio Installer (included with main installer):
+
+- Workloads
+  - **Desktop development with C++**
+  
+#### Project dependencies
+
+- Download and install [Qt](https://www.qt.io/download) for MSVC 2017 32-bit
+  - **Important**: You have to create a personal account
+  - Minimum required version is `5.9`
+- Download and install latest version of [CMake](http://www.cmake.org) for Windows
+  - Make sure to add `cmake` as [global or user environment variable](https://support.shotgunsoftware.com/hc/en-us/articles/114094235653-Setting-global-environment-variables-on-Windows)
+- Download and install latest version of [pandoc](http://www.pandoc.org)
+
+### Project configuration with cmake
+
+Create a subdirectory in TrenchBroom directory called `build`.
+
+#### Visual Studio 2017
+
+Open a command prompt and change directory to `build`:
+
+```bash
+cd <path/to/TrenchBroom>/build
+```
+
+Then, execute this command to configure the project:
+
+```bash
+cmake .. -T v141 -DCMAKE_BUILD_TYPE=Debug -DCMAKE_PREFIX_PATH="C:\Qt\5.13.0\msvc2017"
```

Comment:
I just confirmed `-DCMAKE_BUILD_TYPE=Debug` is not necessary on VS2017. The VS project defaults to Debug, but you can select Release from within VS and both options work.